libraries/monitor/src/logger.js

```diff
@@ -36,6 +36,35 @@ const LEVELS_REVERSE_COLOR = [
   chalk.magenta('DEBUG'),
 ];
 
+const ELIDED = new Set([
+  'credentials',
+  'accessToken',
+  'password',
+  'secretAccessKey',
```

Comment:
how about `secret` (since that's the name of the content of a secret in the secret service)

Reply:
Anybody else who reviews this, if you have other ideas, please let me know! I think we can safely be relatively strict here.